### PR TITLE
`Development`: Update documentation for modeling exercises

### PR DESCRIPTION
**Pull request by minrows**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Description
Modify the documentation of Modeling exercises, to include a brief description of Apollon

### Steps for Testing
Look at the updated documentation about Modeling Exercises: https://artemis-platform--5997.org.readthedocs.build/en/5997/user/exercises/modeling/

### CODE REVIEW

File: docs/user/exercises/modeling.rst [hunk 1]
```diff
@@ -4,6 +4,16 @@ Modeling exercise
     :local:
     :depth: 3
 
+.. note::
+   - Artemis uses a npm library called `Apollon`_ as its modeling editor.
```
Comment (akesfeden) [inline docs/user/exercises/modeling.rst:8]:
```suggestion
   - Artemis uses an npm library called `Apollon`_ as its modeling editor.
```
I am not 100% sure but a quick Google search shows me `an npm` is more common than `a npm`. Also, `npm package` might be a more suitable term than `npm library` but I think you can use the one which sounds more correct to you.

**Review by akesfeden — APPROVED** [review]
Thanks for incorporating the requested changes, I am approving.

**Review by rriyaldhi — APPROVED** [review]
The additional note looks good to me

**Review by tunargs — APPROVED** [review]
Looks good, approved

**Review by LiaLism — APPROVED** [review]
The updated documentation looks good.

**Pull request merged**